services/config/pmm-managed.yaml

```diff
@@ -32,3 +32,6 @@ services:
       retry_backoff_env: "PERCONA_TEST_TELEMETRY_RETRY_BACKOFF"
       retry_count: 20
       send_timeout: 5s
+  pprof:
+    profile_duration: 30s
+    trace_duration: 10s
```

Comment:
making timeouts configurable doesn't bring any value to customers, it's so rare case when we might need to update these values in the future.

Reply:
moved to constants